tern/utils/metadata.py

```diff
@@ -1,15 +1,14 @@
-'''
-Copyright (c) 2017 VMware, Inc. All Rights Reserved.
-SPDX-License-Identifier: BSD-2-Clause
-'''
+# -*- coding: utf-8 -*-
```

Comment:
This is new. Did the linter want to enforce utf-8 encoding?

Reply:
No, just part of my standard header!  Will remove and add a new issue.

Reply:
BTW, it's more a hint for editors, not sure if Python respects it.

Reply:
Apparently the interpreter does per [PEP263](https://www.python.org/dev/peps/pep-0263/). It's something I didn't know of and I think it would be a good addition.

Reply:
Yep, just found that and added https://github.com/vmware/tern/issues/197